### ISSUE TITLE
Indexerror list index out of range sur Gulli et W9

### ISSUE THREAD

**Issue by Dim3333** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1046]
Bonjour,

Dernière version de Kodi et Catch Up (stable) sur Fire TV

Depuis ce matin (pas de modification), sur TV en direct, affichage du message d'erreur "Indexerror list index out of range" sur Gulli et W9 (peut être d'autres pas toutes essayées).

Merci.

**Comment by darodi** [comment]
Ticket doublon, voir 
- #1042
- https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/issues/1039
- #1044

**Comment by darodi** [comment]
>  la solution est d'installer la dernière beta ?

Oui simplement

>  Il n'est pas prévu une MAJ de la version stable ?

Voir avec @joaopa00 , je n'utilise que la bêta

**Comment by joaopa00** [comment]
La version stable ne sera mise à jour que lorsque la version 0.2.38 pour Krypton sera acceptée (c'est déjà fait pour Matrix)  par le serveur officiel de Kodi.
Cela prendra encore plusieurs semaines (voir plusieurs mois)
Lorsque cela sera fait, je proposerai une nouvelle version stable (la version 0.2.39)
Autrement dit pour les mois à venir, il faut utiliser la version béta.

**Comment by Dim3333** [comment]
Effectivement, cela fonctionne de nouveau avec la dernière version Beta. Par contre, Catch Up ne se lance pas automatiquement au démarrage de Kodi alors que l'option est bien activé.
Cela fonctionnait parfaitement avec la version stable.

**Comment by Motopomp** [comment]
la version beta me permet de retrouver les chaine du groupe M6, mais du coup je n'ai plus france television

**Comment by almoreau** [comment]
Just installed 0.2.39-beta17, seems to be working fine for all channels of M6 group !

**Comment by Dim3333** [comment]
Hi @almoreau 

The automatic start of Catch UP works in Kodi ?
